scripts/sysman/device.yml

```diff
@@ -184,6 +238,31 @@ returns:
     - $X_RESULT_ERROR_UNKNOWN
         - "There were problems unloading the device driver, performing a bus reset or reloading the device driver."
 
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Reset device extension"
+class: $sDevice
+name: ResetExt
+details:
+    - "Performs a PCI bus reset of the device. This will result in all current device state being lost."
+    - "All applications using the device should be stopped before calling this function."
+    - "If the force argument is specified, all applications using the device will be forcibly killed."
```

Comment:
suggested rewording: " Prior to calling this function, user is responsible for closing applications using the device unless `force` argument is specified"

Reply:
Fixed